lib/acts_as_list/active_record/acts/position_column_method_definer.rb

```diff
@@ -49,6 +52,38 @@ def self.define_class_methods(caller_class, position_column, touch_on_update)
 
       private
 
+      # Update recordset to start at the given position
+      define_singleton_method :shuffle_to do |position|
+        max = nil
+        if ActiveRecord::VERSION::MAJOR < 4
+          unscoped do
+            max = maximum(position_column)
+          end
+        else
+          max = unscope(:where).maximum(position_column)
+        end
+        return unless max # If no records have a position we don't have to do anything.
+
+        swap_position = max + 2 # Move recordset after the last record
+        update_all_with_touch(
+          "#{quoted_position_column} = #{quoted_position_column_with_table_name} + #{swap_position}"
+        )
+
+        return_from = if ActiveRecord::VERSION::MAJOR < 4
+          unscoped do
+            where("#{quoted_position_column_with_table_name} >= #{swap_position}").minimum(position_column)
+          end
+        else
+          unscope[:where].where("#{quoted_position_column_with_table_name} >= #{swap_position}").minimum(position_column)
+        end
+
+        return_position = return_from - position
```

Comment:
We can't rely on the `position` column being called `position`. Use `send(...` instead.

Reply:
That `position` is a block argument